index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Food lover</title>
+  <link rel="stylesheet" href="./styles/style.css"> 
+  <link rel="stylesheet" href="./styles/index.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+</head>
+<body>
+    <header>
+      <img class="floral-nav-divider" src="assets/floral_white_smaller.png" alt="floral divider">
+      <div class="header-text-content">
+        <h1>Welcome to the Food Fansite</h1>
+        <nav>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="pages/gallery.html">Gallery</a></li>
+            <li><a href="pages/facts.html">Facts</a></li>
+          </ul>
+        </nav>
+      </div>
+      <img class="floral-nav-divider" src="assets/floral_white_smaller.png" alt="floral divider">
+    </header>
+    <main class="content">
+      <section>
+        <h2>Explore the world of food</h2>
+        <p>Here, we share our love for all things foods-related. From yummy pictures to fun facts, this is the place for food lovers!</p>
+        <br>
+        <p>Discover diverse flavors from around the world, including <strong>Asian, Mediterranean, Latin American, and European cuisines</strong>, each offering unique dishes and culinary traditions.</p>
+      </section>
+      <section class="cuisine-section">
+        <div class="cuisine-item">
+          <i class="fas fa-fish"></i>
+          <h3>Mediterranean Cuisine</h3>
+          <a href="https://www.tasteatlas.com/map">
+            <p>Known for fresh vegetables, fish and healthy dishes.</p>
+          </a>
+        </div>
+        <div class="cuisine-item">
+          <i class="fas fa-pepper-hot"></i>
+          <h3>Latin American Cuisine</h3>
+          <a href="https://www.tasteatlas.com/map">
+            <p>Known for tacos, burritos and spicy, hearty flavors.</p>
+          </a>
+        </div>
+        <div class="cuisine-item">
+          <i class="fas fa-pizza-slice"></i>
+          <h3>European Cuisine</h3>
+          <a href="https://www.tasteatlas.com/map">
+            <p>Known for rich pasta dishes, cheeses, pastries, and wines.</p>
+          </a>
+        </div>
+        <div class="cuisine-item">
+          <i class="fas fa-drumstick-bite"></i>
+          <h3>Asian Cuisine</h3>
+          <a href="https://www.tasteatlas.com/map">
+            <p>Known for rice dishes, noodles, and bold flavors.</p>
+          </a>
+        </div>
```

Comment:
I recommend replacing these nested `div`s with semantic tags like `article`. Where else across the files do you see `div`s that could be replaced with a semantic tag?